### DISCUSSION TITLE
Auto dark mode

### DISCUSSION THREAD

**Discussion by marchank0** [blinksh/blink#1648] [Ideas]
Add auto dark mode with ability to assign particular theme for light and dark modes respectively  

Ideally it would be nice to have behaviour similar to iOS/macOS

<img width="379" alt="image" src="https://user-images.githubusercontent.com/1183579/204928558-6d3c69e6-72c0-41df-8681-8f068c0f913d.png">

**Comment by brandmail** [comment]
That would be nice indeed, now we have to manually do it by ourself. And the reason why we have a shell, is because we have servers to automate things for us ;). So would be great if this also be automated ;)

**Comment by EinJochel** [comment]
A couple of yeas ago I found this [workaround](https://github.com/blinksh/blink/issues/752#issuecomment-1068429199), but with a caveat... ([Example theme](https://gist.github.com/EinJochel/edb3ea9fb4a16f28b1ca9f2b135f60cc))
